models/shipping-method/src/shipping-method/transformers.ts

```diff
@@ -0,0 +1,61 @@
+import {
+  LocalizedString,
+  Reference,
+  type TReferenceGraphql,
+} from '@commercetools-test-data/commons';
+import { Transformer } from '@commercetools-test-data/core';
+import type { TShippingMethod, TShippingMethodGraphql } from './types';
+
+const transformers = {
+  default: Transformer<TShippingMethod, TShippingMethod>('default', {
+    buildFields: [
+      'localizedName',
+      'localizedDescription',
+      'taxCategory',
+      'zoneRates',
+      'createdBy',
+      'lastModifiedBy',
+    ],
+  }),
+  rest: Transformer<TShippingMethod, TShippingMethod>('rest', {
+    buildFields: [
+      'localizedName',
+      'localizedDescription',
+      'taxCategory',
+      'zoneRates',
+      'createdBy',
+      'lastModifiedBy',
+    ],
+  }),
+  graphql: Transformer<TShippingMethod, TShippingMethodGraphql>('graphql', {
+    buildFields: [
+      'localizedName',
+      'localizedDescription',
+      'taxCategory',
```

Comment:
The GraphQL representation's `taxCategory` field is not a reference, as it's currently defined in `generator.ts`

If I recall correctly, in the past we've allowed the default model to take a whole `taxCategory` in, then construct a reference from it as needed when performing transformations 🤔

Reply:
Addressed with [70b427e](https://github.com/commercetools/test-data/pull/249/commits/70b427eac3f46d402489bee6c2f4e251b4ed84ff)